.github/workflows/build_test.yml

```diff
@@ -0,0 +1,50 @@
+name: Standard Build and Test
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - '.github/workflows/docker_publish.yml'
+
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - '.github/workflows/docker_publish.yml'
+
+
+jobs:
+  BuildTest:
+    runs-on: ubuntu-latest 
+
+    container:
+      image: cyclus/cyclus-deps
+    
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: setup
+        run: |
+          echo "HOME=/github/home/" >> $GITHUB_ENV
+          echo "PATH=$PATH:${HOME}/.local/bin" >> $GITHUB_ENV
+          echo "LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${HOME}/.local/lib" >> $GITHUB_ENV
+      - name: Building Cyclus
+        run: |
+          mkdir -p ${HOME}/.local/lib/python3.7/site-packages/
+          cd ${GITHUB_WORKSPACE}
+          python install.py -j 2 \
+                            --build-type=Release --core-version 999999.999999 \
```

Comment:
Maybe I haven't noticed this before, but what is `core-version` and is this the right one?